book/robot/exercises/01_reflected_inertia.ipynb

```diff
@@ -629,60 +629,13 @@
   },
   {
    "cell_type": "markdown",
-   "id": "f4415ddf",
-   "metadata": {
-    "colab_type": "text",
-    "id": "vohZdqXrkZJH"
-   },
-   "source": [
-    "## Qualitative Problem ##\n",
-    "\n",
-    "Below, we have a question for you:\n",
-    "\n",
-    "**Why are we seeing such consistent behavior of closed-loop dynamics for the gearbox case, as opposed to the direct-drive case?**\n",
-    "\n",
-    "You can answer in words using 3-5 sentences."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "cc7b725e",
-   "metadata": {
-    "colab_type": "text",
-    "id": "-xr2qacVj2kI"
-   },
-   "source": [
-    "## Your Answer\n",
-    "\n",
-    "Submit your response to the above question in the Gradescope 'written submission' section!"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "161443e9",
-   "metadata": {
-    "colab_type": "text",
-    "id": "zPmeRLtJk410"
-   },
+   "id": "a3e1e5eb",
+   "metadata": {},
    "source": [
-    "## How will this notebook be Graded?\n",
-    "\n",
-    "If you are enrolled in the class, this notebook will be graded using [Gradescope](www.gradescope.com). You should have gotten the enrollement code on our announcement in Piazza.\n",
+    "## Gradescope Verification\n",
     "\n",
-    "For submission of this assignment, you must do two things.\n",
-    "- Download and submit the notebook `reflected_inertia.ipynb` to Gradescope's notebook submission section, along with your notebook for the second problem.\n",
-    "- Copy and Paste your answer to the qualitative problem to Gradescope's written submission section.\n",
-    "\n",
-    "We will evaluate the local functions in the notebook to see if the function behaves as we have expected. For this exercise, the rubric is as follows:\n",
-    "- [5 pts] `pendulum_with_motor_dynamics` must be implemented correctly\n",
-    "- [5 pts] You must answer correctly on why the gearbox and direct-drive cases show different qualitative behavior."
+    "Please answer the questions on Gradescope for this section"
```

Comment:
I think we usually restate the gradescope questions in the notebook, so that way the notebook can be useful for students/profs who use the book without being part of the class. A good reference is Q1 of PSET5 in Gradescope which maps to Question 6.1 in the textbook and its associated notebook.